src/Relation/Binary/ExtensionalEquivalence.agda

```diff
@@ -0,0 +1,37 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Definition of extensional equivalence.
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+open import Level
+open import Relation.Binary.Bundles
+open import Relation.Binary.Definitions
+
+module Relation.Binary.ExtensionalEquivalence
```

Comment:
This should probably live in `Function.Relation.Binary.Equality` (a bit of a mouthful I know, there's discussions to improve the name).

We should also generalise it over setoid equality for both the input and the output of the function!

Reply:
Yes, sorry, should have made clearer that it would be a new module!

Reply:
Okay, I created that new module.